content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
```

Comment:
@geeknoid will ask you to replace all PR URLs with Issue URLs.  If there is no issue, then just describe the fix but don't include a URL.

Reply:
+1 i like that.  asking user to read a PR to figure out things is not friendly.

Reply:
In that logging case, I think the best is to remove the line altogether.
```suggestion
```